cmd/cmd.go

```diff
@@ -66,8 +66,17 @@ func distroSupportsReExec() bool {
 	if !release.OnClassic {
 		return false
 	}
-	switch release.ReleaseInfo.ID {
-	case "fedora", "centos", "rhel", "opensuse", "suse", "poky", "arch", "manjaro", "lirios":
+	var contains = func(haystack []string, needle string) bool {
```

Comment:
Can you please move this to `strutil.Contains`?

Reply:
Hmm, apparently this already exist as `strutil.ListContains`, I'll just reuse that.

Reply:
Ah, perfect.